draft-ietf-ppm-dap.md

```diff
@@ -1714,6 +1710,29 @@ both of which can violate privacy ({{replay-protection}}). Before committing to
 an output share, the Aggregators check whether its report ID has already been
 aggregated and whether the batch bucket being updated has been collected.
 
+### Eager Aggregation {#eager-aggregation}
+
+In general, aggregation cannot begin until the Collector specifies a query and
+an aggregation parameter. However, depending on the VDAF and batch mode in use,
+it is often possible to begin aggregation as soon as reports arrive.
+
+For example, Prio3 has just one valid aggregation parameter (the empty string),
+and so allows for eager aggregation. Both the time-interval and leader-selected
+batch modes defined in this document ({{batch-modes}}) allow for eager
+aggregation, but future batch modes might preclude it.
```

Comment:
This sentence ("Both the time-interval, ...") doesn't really belong here. Perhaps move to a new paragraph below as part of https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/684/files#r2054422312.

Reply:
This paragraph discusses how VDAF and batch mode affect the possibiity of eager aggregation, following directly from the previous paragraph. It makes sense to me.